code/modules/vore/eating/simple_animal_vr.dm

```diff
@@ -75,3 +75,27 @@ mob/living/simple_animal/custom_emote()
 mob/living/simple_animal/say()
 	if (away_from_players()) return
 	. = ..()
+
+/mob/living/simple_animal/attackby(var/obj/item/O, var/mob/user)
+	if (istype(O, /obj/item/weapon/newspaper) && !(ckey || (hostile && faction != user.faction)) && isturf(user.loc))
+		if (retaliate && prob(vore_pounce_chance/2)) // This is a gamble!
+			user.Weaken(5) //They get tackled anyway whether they're edible or not.
+			user.visible_message("<span class='danger'>\the [user] swats \the [src] with \the [O] and promptly gets tackled!</span>!")
+			if (will_eat(user))
+				stop_automated_movement = 1
+				animal_nom(user)
```

Comment:
Should this be animal_nom? What about react_to_attack instead? COMMENCE THE MAUL. Otherwise just turning off mob_vore protects you from this entirely.

Reply:
They still tackle you to the ground whether you're edible or not. Wasn't sure if murder-to-death would be a proportionate response.
I mean, could make it call punchtarget() and just turn around and slap them _once_. A swat for a swat.

Reply:
animal_nom does not do any tackle things, I'm not sure what you're talking about.

Reply:
the tackle is four lines up, before the if(will_eat) check.

Reply:
o